Scripts/variable_dictionary.m

```diff
@@ -5,6 +5,9 @@
 %   and trailer parameters that are loaded and used in the simulation (Simulink).
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% No. Manuever
```

Comment:
%% Maneuver number 

Use %% to split code in sections. This in order to document easily in the future